css/map.css

```diff
@@ -37,13 +37,15 @@ h3{
 }
 
 .entrance{
-  background-color: orange;
+  background-color: black;
```

Comment:
Is it possible to make the entrance black only after the player initially steps off of the space? I was kind of confused at first